diagnostics/model_test/verify_model.py

```diff
@@ -2,10 +2,39 @@
 import argparse
 import numpy as np
 import ngraph
+from google.protobuf import text_format
 import json
 import os
 
 
+def createFolder(directory):
+    try:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+    except OSError:
+        print('Error: Creating directory. ' + directory)
+    os.chdir(output_folder)
+
```

Comment:
did u mean `directory` instead of `output_folder`

Reply:
Yes, you're right. Actually let me move that line outside of createFolder().
It will be like this.
   # Get log file name to save output
    log_file = parameters["log_file_name"]
    output_folder = device1+"-"+device2+"-"+quantized_mode
    createFolder(output_folder)
    **os.chdir(output_folder)**